src/components/PeopleFilters.tsx

```diff
@@ -27,66 +70,76 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <SearchLink
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={cn('button mr-1', {
+                'is-info': centuries.includes('16'),
+              })}
+              params={{ centuries: handleCenturiesChange('16') }}
             >
               16
-            </a>
+            </SearchLink>
 
```

Comment:
make sense to move 16, 17, 18, 19 and 20 to array and render `<SearchLink ` inside cycle for this array, because code is duplicated right now